g2p/mappings/create_fallback_mapping.py

```diff
@@ -8,8 +8,9 @@
 from g2p.mappings.create_ipa_mapping import align_inventories
 from g2p.mappings.utils import is_ipa, unicode_escape
 
+dummy_inventory = ["ɑ", "i", "u", "t", "s", "n"]
```

Comment:
```suggestion
DUMMY_INVENTORY = ["ɑ", "i", "u", "t", "s", "n"]
```

Reply:
There are two other places in the `align_to_dummy_fallback` where we'll have to rename `dummy_inventory` to `DUMMY_INVENTORY`

Reply:
More Python conventions I'm not familiar with yet. Sure, I'm happy to follow PEP8 on this too. I wondered if flake8 would have caught this, but to my surprise I just checked and it doesn't complain. Questions on StackOverflow suggest pylint would complain, but when I try it doesn't. 
No matter, this is clearly a constant and should be upper cased, I will accept the change.